src/main/java/com/doyak/reflector/payload/code/status/ErrorStatus.java

```diff
@@ -12,12 +12,16 @@
 @AllArgsConstructor
 public enum ErrorStatus implements BaseErrorCode {
 
+	// 공통 
     INTERNAL_SERVER_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "COMMON500", "서버 에러입니다."),
     BAD_REQUEST(HttpStatus.BAD_REQUEST, "COMMON400", "잘못된 요청입니다."),
     UNAUTHORIZED(HttpStatus.UNAUTHORIZED, "COMMON401", "인증이 필요합니다."),
     FORBIDDEN(HttpStatus.FORBIDDEN, "COMMON403", "금지된 요청입니다."),
 	
-    // JWT 토큰 관련 에러 
+    // Post
+	POST_NOT_FOUND(HttpStatus.NOT_FOUND, "POST404", "게시글을 찾을 수 없습니다."),
```

Comment:
```suggestion
	POST_NOT_FOUND(HttpStatus.NOT_FOUND, "POST4001", "게시글을 찾을 수 없습니다."),
```

404는 원래 HTTP 표준 상태 코드라서 혼동이 생길 수 있으니 도메인 전용 에러 코드는 1000단위로 구분해서 관리하는 것이 어떨까요?

Reply:
> 404는 원래 HTTP 표준 상태 코드라서 혼동이 생길 수 있으니 도메인 전용 에러 코드는 1000단위로 구분해서 관리하는 것이 어떨까요?

구분할 수 있도록 관리하는거 좋을 것 같아요! 그런데 저희 프로젝트가 대규모가 아니기도 하고, 1씩 커져가기만 하면 프론트 개발할 때 헷갈릴 것 같은데 아래처럼 HTTP 상태코드에 도메인별 코드를 더하는건 어떨까요? 저번 에러코드 올리셨을 때 리뷰 제대로 못남겨서 죄송합니다ㅠㅠ
```suggestion
	POST_NOT_FOUND(HttpStatus.NOT_FOUND, "POST4041", "게시글을 찾을 수 없습니다."),
```

Reply:
저는 저희 코드에서 이미 HTTP 상태코드를 따로 보내서 요청 실패 여부를 판단할 수 있기 때문에, 도메인 코드에 HTTP 상태코드를 붙이면 오히려 혼동될 것 같다고 생각합니다... 예를 들어, post 객체에서 404 상태코드를 사용하는 에러가 9개 이상 존재한다면(물론 말씀대로 대규모 프로젝트가 아니라 이럴 경우가 희박하긴 하나, 추후 확장성을 생각하면 9개가 그리 큰 숫자도 아니라고 생각합니다 ...) 처리하기가 애매해질 것 같아요. 현재 코드에서 이미 HTTP 상태코드와 도메인 코드를 분리해 두었기 때문에, 프론트에서는 요청 실패 자체는 HTTP 상태코드로 판단하고, 구체적인 UI 처리가 필요할 때만 도메인 코드를 사용하는 방식으로 처리해도 괜찮을 것 같습니다!

Reply:
> 저희 코드에서 이미 HTTP 상태코드를 따로 보내서 요청 실패 여부를 판단할 수 있기 때문에, 도메인 코드에 HTTP 상태코드를 붙이면 오히려 혼동될 것 같습니다. 예를 들어, post 객체에서 404 상태코드를 사용하는 에러가 9개 이상 존재한다면(물론 말씀대로 대규모 프로젝트가 아니라 이럴 경우가 희박하긴 하나, 추후 확장성을 생각하면 9개가 그리 큰 숫자도 아니라고 생각합니다 ...) 처리하기가 애매해질 것 같아요. 현재 코드에서 이미 HTTP 상태코드와 도메인 코드를 분리해 두었기 때문에, 프론트에서는 요청 실패 자체는 HTTP 상태코드로 판단하고, 구체적인 UI 처리가 필요할 때만 도메인 코드를 사용하는 방식으로 처리해도 괜찮을 것 같습니다.

좋아요. 말씀하신대로 구체적인 UI 처리가 필요하게 되면 그때 생각해봅시다. 적용할게요!